src/react/backend/location/LocationDetails/storageOptions.js

```diff
@@ -32,6 +32,17 @@ export const storageOptions: ({ [name: string]: any }) = {
         checkCapability: 'locationTypeS3Custom',
         ingestCapability: 's3cIngestLocation',
     },
+    'location-scality-artesca-s3-v1': {
+        name: 'Scality Artesca S3',
+        short: 'ARTESCA',
+        formDetails: LocationDetailsAwsCustom,
+        supportsVersioning: true,
+        supportsReplicationTarget: true,
+        supportsReplicationSource: true,
+        hasIcon: false,
+        checkCapability: 'locationTypeS3Custom',
+        ingestCapability: 's3cIngestLocation',
```

Comment:
I will remove this property since we do not support ingestion from Artesca S3 location type.